addons/project/models/project_project.py

```diff
@@ -438,10 +438,28 @@ def create(self, vals_list):
             for vals in vals_list:
                 if 'label_tasks' in vals and not vals['label_tasks']:
                     vals['label_tasks'] = task_label
+        if len(self.env.companies) > 1 and self.env.user.has_group('project.group_project_stages'):
+            # Select the stage whether the default_stage_id field is set in context (quick create) or if it is not (normal create)
+            stage = self.env['project.project.stage'].browse(self._context['default_stage_id']) if 'default_stage_id' in self._context else self._default_stage_id()
+            # The project's company_id must be the same as the stage's company_id
+            if stage.company_id:
+                vals_list[0]['company_id'] = stage.company_id.id
```

Comment:
why only the firstone?

Reply:
I used to be confused about the multi create decorator. Now I understand how it works but I didn't pay attention to that :sweat: oopsie